verbit/streaming_client.py

```diff
@@ -136,32 +138,36 @@ def socket_timeout(self, timeout: typing.Union[None, float]):
     # Interface #
     # ========= #
     def start_stream(self,
+                     ws_url: str,
                      media_generator: typing.Iterator[bytes],
                      media_config: MediaConfig = None,
                      response_types: ResponseType = ResponseType.Transcript) -> typing.Iterator[typing.Dict]:
         """
         Start streaming media and get back speech recognition responses from server.
 
+        :param ws_url: websocket url to use.
```

Comment:
```suggestion
        :param ws_url: websocket url to use, as obtained from the Ordering API.
```

Reply:
Not sure about the phrasing, but I think it should be clear that the URL needs to be obtained from the Ordering system.

Reply:
done with the suggested phrasing, also made other occurrences uniform.